lib/conncache.c

```diff
@@ -199,7 +199,7 @@ static bool conncache_add_bundle(struct conncache *connc,
 {
   void *p = Curl_hash_add(&connc->hash, key, strlen(key), bundle);
 
-  return p?TRUE:FALSE;
+  return p;
```

Comment:
The function returns a bool value so it's not unnecessary.

Reply:
yeah right but still , we need that ?

Reply:
Yes, because p is a pointer.

Reply:
shown [in this build](https://ci.appveyor.com/project/curlorg/curl/builds/42379683/job/2v795lcec49kf9g0)
~~~
2>.\conncache.c(202) : error C2440: 'return' : cannot convert from 'void *' to 'bool'